src/BundleAnalyzerPlugin.js

```diff
@@ -48,9 +49,9 @@ class BundleAnalyzerPlugin {
       }
 
       if (this.opts.analyzerMode === 'server') {
-        actions.push(() => this.startAnalyzerServer(stats.toJson()));
+        actions.push(() => this.startAnalyzerServer(stats.toJson(), bundleDir));
       } else if (this.opts.analyzerMode === 'static') {
-        actions.push(() => this.generateStaticReport(stats.toJson()));
+        actions.push(() => this.generateStaticReport(stats.toJson(), bundleDir));
```

Comment:
I'm not a huge fan in how I had to add another argument to these methods to get this working